docs/compression_algorithms/Pruning.md

```diff
@@ -82,15 +94,25 @@ sparsity and filter pruning algorithms. It can be enabled by setting a non-zero
         "num_init_steps": 3, // Number of epochs for model pretraining before starting filter pruning. 0 by default.
         "pruning_steps": 10, // Number of epochs during which the pruning rate is increased from `pruning_init` to `pruning_target` value.
         "filter_importance": "L2", // The type of filter importance metric. Can be one of `L1`, `L2`, `geometric_median`. `L2` by default.
+        "interlayer_ranking_type": "unweighted_ranking", // The type of filter ranking across the layers. Can be one of `unweighted_ranking`, `learned_ranking`. `unweighted_ranking` by default.
         "all_weights": false, // Whether to prune layers independently (choose filters with the smallest importance in each layer separately) or not. `False` by default.
         "prune_first_conv": false, // Whether to prune first Convolutional layers or not. First means that it is a convolutional layer such that there is a path from model input to this layer such that there are no other convolution operations on it. `False` by default.
         "prune_last_conv": false, // Whether to prune last Convolutional layers or not.  Last means that it is a Convolutional layer such that there is a path from this layer to the model output such that there are no other convolution operations on it. `False` by default.
         "prune_downsample_convs": false, // Whether to prune downsample Convolutional layers (with stride > 1) or not. `False` by default.
         "prune_batch_norms": true, // Whether to nullifies parameters of Batch Norm layer corresponds to zeroed filters of convolution corresponding to this Batch Norm. `True` by default.
         "zero_grad": true // Whether to setting gradients corresponding to zeroed filters to zero during training, `True` by default.
+        "save_ranking_coeffs_path": "path/coeffs.json", // Path to save .json file with interlayer ranking coefficients.
+        "load_ranking_coeffs_path": "PATH/learned_coeffs.json", // Path to loading interlayer ranking coefficients .json file, pretrained earlier.
+        "legr_params": { // Set of parameters, that can be set for 'learned_ranking' interlayer_ranking_type case
+            "generations": 200, //  Number of generations for evolution algorithm optimizing. 400 by default
+            "train_steps": 150, // Number of training steps to estimate pruned model accuracy. 200 by default 
+            "max_pruning": 0.6, // Pruning level for the model to train LeGR algorithm on it. If learned ranking will be used for multiple pruning rates, the highest should be used as `max_pruning`. If model will be pruned with one pruning rate, target pruning rate should be used.
+            "random_seed": 42, // Random seed for ranking coefficients generation during optimization 
+        },
```

Comment:
Do we need to remove "Export pruning algorithm description:" section from this document? As we know the corresponding OV transformation is in progress but we don't use this NNCF functionality anyway.

Reply:
I think this section should be removed/rewritten only when the export will be fully replaced from NNCF.

Reply:
Do we use this anywhere besides the tests? If not, I don't see any reason why we should keep it.

Reply:
The section was deleted.